README.md

```diff
@@ -289,15 +305,15 @@ Note: On 64-bit computers, the `cpu_smp_processor_affinity_mask` property contai
 
 #### `identity_type`
 
-Specifies the account identity under which the application pool runs. Note: Starting in IIS 7.5 the default value is 'ApplicationPoolIdentity'. (In IIS 7.0 the default value was 'NetworkService').  Valid options 'ApplicationPoolIdentity', 'LocalService', 'LocalSystem', 'NetworkService', or 'SpecificUser'.
+Specifies the account identity under which the application pool runs. Note: Starting in IIS 7.5 the default value is `ApplicationPoolIdentity`. (In IIS 7.0 the default value was `NetworkService`).  Valid options `ApplicationPoolIdentity`, `LocalService`, `LocalSystem`, `NetworkService`, or `SpecificUser`.
 
 #### `idle_timeout`
 
 Specifies how long (in minutes) a worker process should run idle if no new requests are received and the worker process is not processing requests. After the allocated time passes, the worker process requests that it be shut down by the WWW service.
 
 #### `idle_timeout_action`
 
-Specifies the action to perform when the `idle_timeout` duration has been reached. Before IIS 8.5, a worker process that was idle for the duration of the `idle_timeout` property would be terminated. After IIS 8.5, you have the choice of terminating a worker process that reaches the `idle_timeout` limit, or suspending it by moving it from memory to disk. Suspending a process will likely take less time and consume less memory than terminating it. Valid options 'Terminate' or 'Suspend'.
+Specifies the action to perform when the `idle_timeout` duration has been reached. Before IIS 8.5, a worker process that was idle for the duration of the `idle_timeout` property would be terminated. After IIS 8.5, you have the choice of terminating a worker process that reaches the `idle_timeout` limit, or suspending it by moving it from memory to disk. Suspending a process will likely take less time and consume less memory than terminating it. Valid options `Terminate` or `Suspend`.
```

Comment:
Valid options for this are both passed as strings, so should be in single quotes